template/src/components/dialog-about/index.js

```diff
@@ -67,6 +68,16 @@ const About = (props) => {
         >
           {`Version v${appVersion}. Powered by WebCatalog.`}
         </Typography>
+        <Typography variant="body2" className={classes.version}>
+          {versions.map(({ name, version }) => (
+            <div>
+              {name}
+              :
+              {' '}
+              {version}
+            </div>
```

Comment:
`<Typography>` is `<p>`, so it doesn't like that you have `<div>` as its children.